### PR TITLE
Update waku nodes

### PR DESCRIPTION
**Pull request by cammellos**
status:ready

### CODE REVIEW

File: src/status_im2/contexts/profile/create/events.cljs [hunk 1]
```diff
@@ -28,22 +20,3 @@
            :password           (native-module/sha3 (security/safe-unmask-data password))
            :imagePath          (profile.config/strip-file-prefix image-path)
            :customizationColor color})})
-
-(rf/defn login-new-profile
```
Comment (flexsurfer) [inline src/status_im2/contexts/profile/create/events.cljs:32]: hm

**Review by flexsurfer — APPROVED** [review]

**Comment by flexsurfer** [comment]
is this a replacement for https://github.com/status-im/status-mobile/pull/17418/files ?

File: src/status_im2/contexts/profile/create/events.cljs (hunk 1 above)
Comment (cammellos) [reply src/status_im2/contexts/profile/create/events.cljs:32]: it's not used as far as I can see?

**Comment by cammellos** [comment]
> is this a replacement for https://github.com/status-im/status-mobile/pull/17418/files ?

Yes, it includes it

File: src/status_im2/contexts/profile/create/events.cljs (hunk 1 above)
Comment (flexsurfer) [reply src/status_im2/contexts/profile/create/events.cljs:32]: yeah, just wondering why, probably i forgot to remove it during refactoring

**Comment by yakimant** [comment]
@cammellos, please tell me, when https://github.com/status-im/status-go/pull/4063 is good to merge

**Comment by cammellos** [comment]
@yakimant I have also cherry picked (or rather copied some code) from that PR, so that one as well can be closed, since we required some further changes to make it work correctly (it would have only worked for new users, but not for users that were created before that PR, since config is stored in the database)

**Comment by churik** [comment]
@cammellos Unfortunately sometimes messages are not being delivered (in the community I saw > 10 mins, after opening chat view messages that were sent at 11:43 were delivered at 11:58)

 3 messages: image
Logs of sender: https://drive.google.com/file/d/1_p1IF3FaAZ-v7C5KcPeuq_wO7fXfe878/view?usp=sharing (

Logs of receiver: [logs (61).zip](https://github.com/status-im/status-mobile/files/12763510/logs.61.zip)

Cannot find specific steps to reproduce - in my case messages were sent immediately after the receiver device (IOS) was getting back after the background.
Sender device: status desktop master

**Comment by pavloburykh** [comment]
@cammellos also we have found issues related to new counters logic. I have re-opened an issue and described current bugs in this comment https://github.com/status-im/status-mobile/issues/16696#issuecomment-1740705191

**Comment by churik** [comment]
And all e2e from here are failing again because of the message reliability 
@cammellos @siphiuel tell me if you need more logs/cases

**Comment by yakimant** [comment]
@churik, @cammellos, is anything blocking this PR from being merged? Or those are separate issues?

**Comment by churik** [comment]
@yakimant 

we need to identify the reason for message reliability issues.

So it will be likely to merge https://github.com/status-im/status-mobile/pull/17344, rebase status-go branch onto the version with fix and retest this one

I'm on testing https://github.com/status-im/status-mobile/pull/17344

**Comment by pavloburykh** [comment]
Hello @cammellos ! We have finally merged messages reliability fix into develop. Could you please rebase go and mobile branches once again so we perform another round of testing? Thank you!

**Comment by pavloburykh** [comment]
@cammellos thank you! PR is ready for merge.

**Comment by pavloburykh** [comment]
e2e fails are not PR related. Ready for merge.

**Pull request merged**